test/system/react_components/iterations_summary_table_test.rb

```diff
@@ -0,0 +1,19 @@
+require "application_system_test_case"
```

Comment:
For each component, we can have a system test like this which tests functionality.